dev/com.ibm.ws.javaee.ddmodel/resources/com/ibm/ws/javaee/internal/ddmodel/resources/DDModelMessages.nlsprops

```diff
@@ -70,21 +83,94 @@ invalid.href.prefix=CWWKC2260E: The href attribute value of the {2} element does
+# invalid.deployment.descriptor.namespace.useraction=Verify and correct the version or namespace in the root element of the deployment descriptor, or verify and correct the features in the server configuration.
 
-invalid.deployment.descriptor.version=CWWKC2263E: The version {2} attribute specified in the {0} deployment descriptor on line {1} is not valid.
-invalid.deployment.descriptor.version.explanation=An invalid version was specified in the root element of the deployment descriptor.
-invalid.deployment.descriptor.version.useraction=Verify and correct the version number in the root element of the deployment descriptor.
+# REPLACED BY:
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the deployment descriptor
+#    {2} is a namespace value within the deployment descriptor
+# -----------------------------------------------------------------------
+unsupported.descriptor.namespace=CWWKC2262E: The deployment descriptor {0}, at line {1}, specifies unsupported namespace {2}.
+unsupported.descriptor.namespace.explanation=The namespece which was specified in the deployment descriptor is not valid for that type of deployment descriptor.
+unsupported.descriptor.namespace.useraction=Correct the namespace specified in the deployment descriptor.
+
+# ADDED:
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the deployment descriptor
+#    {2} is a namespace value within the deployment descriptor
+# -----------------------------------------------------------------------
+unsupported.descriptor.namespace=CWWCK27780E: The deployment descriptor {0}, at line {1}, specifies unsupported namespace {2}.
+unsupported.descriptor.namespace.explanation=The deployment descriptor has an incorrect namespace.
+unsupported.descriptor.namespace.useraction=Correct the namespace to a valid value.
+
+# ADDED:
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the deployment descriptor
+#    {2} is a version value within the deployment descriptor
+#    {3} is a namespace value within the deployment descriptor
+#    {4} is the correct namespace value which should be present in the deployment descriptor
+# -----------------------------------------------------------------------
+unsupported.descriptor.namespace.for.version=CWWCK27781W: The deployment descriptor {0}, at line {1}, specifies version {2} and namespace {3}, but should have namespace {4}.
```

Comment:
Change "deployment descriptor {0}" to "{0} deployment descriptor"

Reply:
Updated.